cocos2d/cocos2d-objc
# Issue #1071 test case.

Reusable openGLView fixed it.

How to test the test:
1) Run mac FullScreenTest and change to third test
2) Ensure that you're in fullscreen
3) Press load something async. It loads normally in 1.0 develop [SUCCESS] and it loads a white square in 0.99.5 [FAILURE]

Issue proof with failing test: 
https://github.com/psineur/cocos2d-iphone/tree/0.99.5-issue1071-proof
